src/services/oauth2/hooks/index.js

```diff
@@ -4,7 +4,7 @@ const { Forbidden, MethodNotAllowed } = require('../../../errors');
 const globalHooks = require('../../../hooks');
 const Hydra = require('../hydra.js');
 
-const properties = 'title="username" style="height: 26px; width: 180px; border: none;"';
+const properties = 'title="username" style="height: 26px; width: 180px; border: none;" loading="lazy"';
```

Comment:
this is currently supported only by a few browsers
https://caniuse.com/?search=loading%3D%22lazy%22

Reply:
then it will improve it for those browsers who support it. but, it is not obligatory for the others and wont't prevent the site from working.